tests/conftest.py

```diff
@@ -187,3 +188,22 @@ def copr_client_mock(get_list_return=None):
 
     copr_mock = flexmock(mock_chroot_proxy=flexmock(get_list=lambda: get_list_return))
     return copr_mock
+
+
+def configure_git():
```

Comment:
I'd prefer this being a fixture set for the whole test session: https://docs.pytest.org/en/stable/fixture.html#autouse-fixtures-fixtures-you-don-t-have-to-request